README.md

```diff
@@ -31,6 +32,8 @@ For deployment of NMO using these images you need:
 
 Then run `operator-sdk run bundle quay.io/medik8s/node-maintenance-operator-bundle:latest`
 
+### Deploy the last release version
+Click on `Install` in the Node Maintenance Operator page under [OperatorHub.io](https://operatorhub.io/operator/node-maintenance-operator), and follow it's instructions (to install the operator-sdk and the operator).
```

Comment:
- instructions install OLM not operator-sdk
- please add a new line

Reply:
Do you mean?
```suggestion
- Click on `Install` in the Node Maintenance Operator page under [OperatorHub.io](https://operatorhub.io/operator/node-maintenance-operator)
- Follow the instructions to install [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/)
```

Reply:
about OLM yes (but keep that it installs the operator as well)
the new line is missing after this though (an empty line between this and the next heading line)

Reply:
How about now?